internal/api.go

```diff
@@ -0,0 +1,35 @@
+package api
```

Comment:
Not a huge fan of this name. It seems project like geth have a more generalized API (rpc, websockets, ipc, etc), but in our case this is just RPC (for now).

Reply:
Look in geth's `internal/ethapi` package I believe its`ethapi`